generators/app/templates/src/services/AnalyticsService.js

```diff
@@ -3,8 +3,7 @@ import { GoogleAnalytics } from 'redux-beacon/targets/google-analytics';
 
 const pageView = action => ({
   hitType: 'pageview',
-  page:
-    action.payload.pathname === '/' ? 'accountstatus' : action.payload.pathname
+  page: action.payload.pathname === '/' ? 'accountstatus' : action.payload.pathname
```

Comment:
Use `route` constants here

Reply:
This PR only changes this because of the linter autofix, but I'll fi it anyway